docs/setup_installation/aws/restrictive_permissions.md

```diff
@@ -269,6 +269,8 @@ Hopsworks put its logs in Amazon CloudWatch so that you can access them without
     }
 ```
 ### Upgrade permissions 
+!!! warning
```

Comment:
I think we should just remove the section.

Reply:
done, removed